### ISSUE TITLE
SyntaxError: Use of const in strict mode.

### ISSUE THREAD

**Issue by wmarra** [vially/cpanel-lib#5]
```
|=> unguedes [~/Sites/build-store] (master) nodemon app.js
9 Nov 12:47:44 - [nodemon] 1.7.1
9 Nov 12:47:44 - [nodemon] to restart at any time, enter `rs`
9 Nov 12:47:44 - [nodemon] watching: *.*
9 Nov 12:47:44 - [nodemon] starting `node app.js`
/Users/unguedes/Sites/build-store/node_modules/cpanel-lib/node_modules/qs/lib/index.js:5
const Stringify = require('./stringify');
^^^^^
SyntaxError: Use of const in strict mode.
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/unguedes/Sites/build-store/node_modules/cpanel-lib/lib/cpanel.js:1:72)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
9 Nov 12:47:45 - [nodemon] app crashed - waiting for file changes before starting...
```

**Comment by vially** [comment]
Duplicate of #4
